src/util/subprocess.h

```diff
@@ -1353,6 +1357,11 @@ namespace detail {
       if (stream.err_write_ != -1 && stream.err_write_ > 2)
         close(stream.err_write_);
 
+#ifdef CODE_COVERAGE_ENABLED
+      __gcov_dump();
+      __gcov_reset();
```

Comment:
The code in https://inbox.sourceware.org/gcc-cvs/20200505141638.8B852388F43D@sourceware.org/T/ suggests that `__gcov_dump()` should be called before exec() because otherwise a successful exec will cause the information to be discarded, and `__gcov_reset()` should be called after exec() returns, because that means exec failed and the information wasn't discarded, so coverage stats should continue to be gathered without duplicating the info that's already been dumped. Doing the reset unconditionally prior to the exec function this way seems like a reasonable alternative approach to that to me.

Reply:
> Doing the reset unconditionally prior to the exec function this way seems like a reasonable alternative approach to that to me.

It looks better to me because it counts the failed `execvp()`, which returns.